whats_fresh/whats_fresh_api/templates/entry.html

```diff
@@ -3,38 +3,47 @@
 {% block content %}
 <div class='entry'>
     <a href='{% url 'list-vendors-edit' %}'>
-        <h4>Vendors</h4>
+        <!-- <h4>Vendors</h4> -->
+        <h4>{{ theme|get_fieldname:'vendors' }}</h4>
```

Comment:
I think this is what I mentioned on IRC -- you'll need to load the tag inside of each file you want to use it in, so here in entry.html.